io-engine/src/bdev/nvmx/controller.rs

```diff
@@ -911,6 +911,7 @@ pub(crate) async fn destroy_device(name: String) -> Result<(), BdevError> {
             // Release the lock before waiting for controller shutdown.
             drop(controller);
 
+            // clippy bug...
```

Comment:
Is this comment needed?

Reply:
Oh no, I was using wrong clippy I think, then reverted the change but left the comment